# True wind direction and speed. Direction is in degrees, 0 = east, increases counter-clockwise
# speed is m/s (?)
int32 direction
int32 speed
